Editor/LocalisationEditor.cs

```diff
@@ -60,7 +60,7 @@ private void OnGUI()
 			if (localSchema == null)
 			{
 				EditorGUILayout.HelpBox(
-					"A key schema is a list of all the localisation keys in the project - generate a new one and populate it with all the keys you need, or drag the existing asset into the field above.",
+					"A key schema is a list of all the localisation keys in the project - generate a new one with the Assets menu and populate it with all the keys you need, or drag the existing asset into the field above.",
```

Comment:
I think the grammatically correct is `using the Assets menu` :neckbeard:

Reply:
Modified